### PR TITLE
Made right click menu 'Track task' untrack on task that are tracked.

### PR DESCRIPTION
**Pull request by Sakelig**
Removed if condition so you can untrack task with right click menu 'Track task'.
Similiar to dobble LMB on task that is tracked.
Similiar to pressing 'Complete task' on the right click menu on a already completed task.

pls tell if im dumb Okayge

Image shows where the change is done and clicking it will now untrack the task
image

### CODE REVIEW

File: src/components/TaskTracker/Task.tsx [hunk 1]
```diff
@@ -66,7 +66,7 @@ export const Task = ({ task, tasks }) => {
   };
 
   const markNotCompleteWhenTracking = () => {
-    if (!task.inProgress) toggleInProgressState(task.id);
+    toggleInProgressState(task.id);
```
Comment (Melkeydev) [inline src/components/TaskTracker/Task.tsx:69]: @justnat3 IIRC you created this implementation - does this look OK to you/

Comment (justnat3) [reply src/components/TaskTracker/Task.tsx:69]: @Sakelig, I seem to remember this bug... Honestly thought this was fixed- 

Your patch does fix the behavior, thank you! I do have one nit-pick. While you are in there could you change the text dependent on the state to "track"/"un-track" - and "complete"/"not-finished"

Cheers,
-Nate

**Comment by Melkeydev** [comment]
@Sakelig are you still interested in merging this?

**Comment by Sakelig** [comment]
> @Sakelig are you still interested in merging this?

Yes 🐸
Ill look into the conflict when i get home and fix 👍

**Comment by Sakelig** [comment]
i dunno, 'toggleInProgressState' got changed so dunno what the issue is now so ¯\_(ツ)_/¯
might come back to this oss at a later point again

**Pull request closed without merging**